src/main/java/org/folio/circulation/support/CqlQuery.java

```diff
@@ -29,6 +29,14 @@
     return Result.of(() -> new CqlQuery(format("%s==\"%s\"", index, value), none()));
   }
 
+  /**
+   * The expression `comparitorSymbol: '<>' ` for finding a non-empty value
+   * is not supported in the CqlQuery.
+   */
+  public static Result<CqlQuery> exactNotEmpty(String index) {
```

Comment:
The JavaDoc suggests this uses `<>` however the implementation appears to use `>`

Please can this be updated. My preference would be to change this to explicitly be a greater than match, e.g `greaterThan(String index, String value)`

Also please can a unit test be added for this.